themes/default/_messages.twig

```diff
@@ -1 +1,6 @@
-                {{ messages | raw }}
\ No newline at end of file
+                {{ messages | raw }}
+                {% if view == 'home' %}
+                <div class="messages information">
+                    <p><a href="http://doc.wallabag.org">wallabag v2 is out, please read the documentation to upgrade.</a></p>
```

Comment:
but why? why not show a message once? Version 2 runs slowly on my device, so it is not needed.